### DISCUSSION TITLE
I made a web interface for sherlock :)

### DISCUSSION THREAD

**Discussion by ghost** [sherlock-project/sherlock#1643] [Show and tell]
Sherlock is no longer just a CLI tool

Access sherlock from the web
here's the GitHub:
https://github.com/therealcyber71/watson-recon
and the implementation
https://www.watsonrecon.tk/

**Comment by hasnatkhan786** [comment]
It took time to load and very slow process[,](https://catchmyswag.com/collections/accessories) but I loved your project just awesome.

**Reply by ghost** [reply]
Hi!
Thanks for trying it out, I'm still working on trying to make it work faster
